hibike/flash.sh

```diff
@@ -1,5 +1,8 @@
 #/bin/bash
-make clean
+
+#make clean doesn't actually work.  Therefore,
+rm -rf ./bin
```

Comment:
This might do bad things if ./flash.sh is accidentally run outside of `PieCentral/hibike` but this probably won't be a major issue.